manifests/charts/istiod-remote/templates/configmap.yaml

```diff
@@ -55,7 +55,7 @@
 {{- $originalMesh := include "mesh" . | fromYaml }}
 {{- $mesh := mergeOverwrite $originalMesh .Values.meshConfig }}
 
-{{- if not .Values.global.centralIstiod }}
+{{- if or (eq .Values.global.externalIstiod true) (eq .Values.global.centralIstiod true) }}
```

Comment:
is eq .... true here needed?  Note the deleted old doesn't check if it equals to true.

Reply:
this file is useless, so removed